### PR TITLE
evms: drop unneeded evmone flags

### PR DESCRIPTION
**Pull request by chfast**

### CODE REVIEW

**Comment by chfast** [comment]
Let me test it a bit first.

File: evms/evmone.go [hunk 1]
```diff
@@ -54,7 +54,8 @@ func (evm *EvmoneVM) Name() string {
 }
 
 func (evm *EvmoneVM) GetStateRoot(path string) (root, command string, err error) {
-	cmd := exec.Command(evm.path, "--ignore-state-root", "--ignore-logs", "--trace", path)
+	// TODO: We must use --trace because evmone does not output state root by default
+	cmd := exec.Command(evm.path, "--trace", path)
```
Comment (holiman) [inline evms/evmone.go:58]: No we must not
```
evmone-statetest ./cases/00000006-naivefuzz-0.json 
Note: Google Test filter = -stCreateTest.CreateOOGafterMaxCodesize:stQuadraticComplexityTest.Call50000_sha256:stTimeConsuming.static_Call50000_sha256:stTimeConsuming.CALLBlake2f_MaxRounds:VMTests/vmPerformance.*:
[==========] Running 1 test from 1 test suite.
[----------] Global test environment set-up.
[----------] 1 test from ./cases
[ RUN      ] ./cases.00000006-naivefuzz-0
/home/martin/workspace/evmone/test/statetest/statetest_runner.cpp:38: Failure
Expected equality of these values:
  logs_hash(get<state::TransactionReceipt>(res).logs)
    Which is: 0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347
  expected.logs_hash
    Which is: 0x0000000000000000000000000000000000000000000000000000000000000000
Google Test trace:
/home/martin/workspace/evmone/test/statetest/statetest_runner.cpp:19: London/0
/home/martin/workspace/evmone/test/statetest/statetest_runner.cpp:46: Failure
Expected equality of these values:
  state::mpt_hash(state.get_accounts())
    Which is: 0xad1024c87b5548e77c937aa50f72b6cb620d278f4dd79bae7f78f71ff75af458
  expected.state_hash
    Which is: 0x0000000000000000000000000000000000000000000000000000000000000000
Google Test trace:
/home/martin/workspace/evmone/test/statetest/statetest_runner.cpp:19: London/0
[  FAILED  ] ./cases.00000006-naivefuzz-0 (0 ms)
[----------] 1 test from ./cases (0 ms total)

[----------] Global test environment tear-down
[==========] 1 test from 1 test suite ran. (0 ms total)
[  PASSED  ] 0 tests.
[  FAILED  ] 1 test, listed below:
[  FAILED  ] ./cases.00000006-naivefuzz-0

 1 FAILED TEST
```
We _can_ parse the stateroot from the output above.

**Review by holiman — APPROVED** [review]
LGTM

File: evms/evmone.go [hunk 2]
```diff
@@ -54,11 +54,17 @@ func (evm *EvmoneVM) Name() string {
 }
 
 func (evm *EvmoneVM) GetStateRoot(path string) (root, command string, err error) {
-	cmd := exec.Command(evm.path, "--ignore-state-root", "--ignore-logs", "--trace", path)
+	cmd := exec.Command(evm.path, "--trace-summary", path)
 	data, err := cmd.CombinedOutput()
```
Comment (holiman) [inline evms/evmone.go:58]: Looking at the reference outputs -- we're only using the stderr aren't we? In that case, no need to use `CombinedOutput()`. But there's no easy `StderrOutput()` handy, so this is fine for now

**Review by holiman — APPROVED** [review]

**Pull request merged**